docs/canvas/canvas-elements.asciidoc

```diff
@@ -0,0 +1,259 @@
+
+. Change the *Appearance* and *Border* options.
+ 
+[float]
+[[apply-element-styles]]
+==== Apply a set of styles
+
+To make your element look exactly the way you want, apply CSS overrides.
+
+. Next to *Element style*, click *+*, then select *CSS*.
+
+. Enter the *CSS*. For example, to center the Markdown element, enter:
++
+[source,js]
+--------------------------------------------------
+.canvasRenderE1 h1 {
+text.align: center;
+}
+--------------------------------------------------
+
+. Click *Apply stylesheet*.
+
+[float]
+[[configure-auto-refresh-interval]]
+==== Change the data auto-refresh interval
+
+Increase or decrease how often your data refreshes on your workpad.
+
+. In the top left corner, click the *Control settings* icon.
+
+. Under *Change auto-refresh interval*, select the interval you want to use.
++
+[role="screenshot"]
+image::images/canvas-refresh-interval.png[Element data refresh interval]
+
+TIP: To manually refresh the data, click the *Refresh data* icon. 
+
+[float]
+[[organize-element]]
+=== Organize the elements on your workpad
+
+Choose where you want the elements to appear on your workpad.
+
+[float]
+[[move-canvas-elements]]
+==== Move elements
+
+Move the element to a preferred location on your workpad. As you move the element, notice the alignment lines that appear to help you place the element exactly where you want it.
+
+* Click and drag the element to your preferred location.
+
+* To move the element by 1 pixel, select the element, then use your arrow keys.
+
+[float]
+[[resize-canvas-elements]]
+==== Resize elements
+
+Make your elements bigger or smaller than the default size.
+
+. Select the element.
+
+. Click and drag the resizing icon to the size you want.
+
+[float]
+[[align-canvas-elements]]
+==== Align elements
+
+Align two or more elements on your workpad.
+
+. Press Shift, then select the elements you want to align.
+
+. Click the *Element options* icon in the top right corner, then select *Align elements*.
+
+. From the *Alignment* menu, select how you want to align the elements on the workpad.
++
+[role="screenshot"]
+image::images/canvas-align-elements.gif[Align elements]
+
+[float]
+[[distribute-canvas-elements]]
+==== Distribute elements
+
+Distribute two or more elements on your workpad.
+
+. Press Shift, then select the elements you want to distribute.
+
+. Click the *Element options* icon in the top right corner, then select *Distribute elements*.
+
+. From the *Distribution* menu, select how you want to distribute the elements on the workpad.
++
+[role="screenshot"]
+image::images/canvas-distribute-elements.gif[Distribute elements]
+
+[float]
+[[change-element-order]]
+==== Change the element order
+
+Change the order of how the elements are displayed on your workpad.
+
+. Select an element.
+
+. In the top right corder, click the *Element options* icon. 
+
+. Select *Order*, then select the order that you want the element to appear.
+
+[float]
+[[element-save]]
+=== Save elements
```

Comment:
Can we add a GIF for this section? I can record one for you if you need.

Reply:
That would be super helpful, thank you!